packages/apollo-voyager-audit/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import { buildPath } from '@aerogear/apollo-voyager-tools'
+import { GraphQLResolveInfo } from 'graphql'
+import { IFieldResolver } from 'graphql-tools'
+import pino from 'pino'
+
+const log = pino()
```

Comment:
[Question] Any reason we use pino instead of plain console.log? Message formatting?

Reply:
We need to log in JSON format for audit logging and we liked pino back then because it logs in JSON by default and it is efficient.
I simply moved over stuff from the old code base.